docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -203,6 +216,10 @@ const DebugPage: React.FunctionComponent<Props> = ({
         )?.pathMapping || '';
     }
 
+    if (urlPath.startsWith('/')) {
```

Comment:
Question: Is there a case that `urlPath` doesn't start with a slash?

Reply:
Hmm I was thinking about `exact:` and relative path (if possible?)

Reply:
I think `exact:` will be stripped by the `extractUrlPath` method above but I'm not sure about the relative path.
So let's leave it as is. We can fix it later if we found any problems. 😉